packages/recipes/src/index.ts

```diff
@@ -63,7 +69,12 @@ export function recipe<Variants extends VariantGroups>(
 
   const config: PatternResult<Variants> = {
     defaultClassName,
+    rawDefaultClassName: defaultClassName.split(' ')[0],
     variantClassNames,
+    // @ts-expect-error
+    rawVariantClassNames: mapValues(variantClassNames, (classNames) =>
```

Comment:
Could we do this work at runtime instead in `createRuntimeFn`? This will avoid extra data being sent to the runtime.